cli/src/commands/init/index.ts

```diff
@@ -271,6 +274,27 @@ export default class Init extends BaseCommand {
     await this.ignoreEnvFile(envFile);
   }
 
+  // New API keys need to be replicated until can be used in a particular region/database
+  async waitUntilAPIKeyIsValid(workspace: string, region: string, database: string) {
+    const xata = await this.getXataClient();
+    const maxRetries = 10;
+    let retries = 0;
+    while (retries++ < maxRetries) {
+      try {
+        await xata.branches.getBranchList({ workspace, region, database });
+        return;
+      } catch (err) {
+        if (err instanceof Error && err.message.includes('Invalid API key')) {
+          this.info('Waiting until the new API key is ready to be used...');
```

Comment:
do we need to tell the user about this? normally this will take 1-2 seconds, maybe only telling them if retries > 2?

Reply:
That's a good point. I agree with you but at the same time I don't want the UI to feel stuck if for some reason the replication takes longer.

Reply:
What about this? https://github.com/xataio/client-ts/pull/769/commits/f816a77a449e7ccabab89dab11196db2da990b2e

Reply:
yeah, I don't know how easy would it be to keep adding dots or similar to make it explicit to the user that the CLI is alive, I'm trying to avoid user questions & uncertainties around this message, but maybe it's not a big deal.

anyway let's not block the PR on this

Reply:
oh, that works! 👍 thank you!